kvdb/postgres/readwrite_tx.go

```diff
@@ -158,11 +158,11 @@ func (tx *readWriteTx) OnCommit(cb func()) {
 }
 
 // QueryRow executes a QueryRow call with a timeout context.
-func (tx *readWriteTx) QueryRow(query string, args ...interface{}) *sql.Row {
-	ctx, cancel := tx.db.getTimeoutCtx()
-	defer cancel()
+func (tx *readWriteTx) QueryRow(query string, args ...interface{}) (*sql.Row,
```

Comment:
Should we get rid of the `Query()` function below as it's currently not used but would present the same problem probably?

Reply:
Good point, removed.

Reply:
+1